verification/dependencies/crypto/subtle/constant_time.gobra

```diff
@@ -18,5 +18,9 @@ trusted
 requires sl.Bytes(x, 0, len(x))
 requires sl.Bytes(y, 0, len(y))
 ensures  len(x) != len(y) ==> res == 0
+ensures res != 0 ==> 
+    unfolding sl.Bytes(x, 0, len(x)) in 
+    unfolding sl.Bytes(y, 0, len(y)) in 
+    forall i int :: { x[i] } { y[i] } 0 <= i && i < len(x) ==> x[i] == y[i] 
```

Comment:
Please use the `GetByte` function defined in the package `verification/utils/slices/slices.gobra` to avoid the unfoldings here